### DISCUSSION TITLE
Hosting Vvveb with Coolify

### DISCUSSION THREAD

**Discussion by mraf** [givanz/Vvveb#185] [Ideas]
Hello @givanz ,

I noticed you browsing various ideas for hosting vvveb. So far I found the quickest way to deploy whole app is coolify https://coolify.io/self-hosted 

You can install it via Dockerfile, Docker Compose or Exsisting Docker image from Docker Hub

3rd option is the quickest one with vvveb/vvvebcms:latest image. Whole deploy process is taking under 1min with database setup. 

1. Create database. You can select Mysql, MariaDB and they just works great. Postgress have some connection issues
2. Deploy from `vvveb/vvvebcms:latest` Docker image with persistent storage and `/var/www/html` as destination path

and that's it. Multisite domains also working with no issue :) 

Coolify have also pre built docker templates which I think are worth to check or maybe even promote vvveb there.

All setup is very smooth and I think can be very good option for people who have issues with their hostings.

**Comment by givanz** [comment]
Hi

Thank for suggestion, I added a pull request with a predefined template https://github.com/coollabsio/coolify/pull/3219 to make it even easier to deploy.

I will check installation with postgres to see why is not working properly with docker.

Thanks for feedback.

**Reply by mraf** [reply]
vvveb has been merged now! cool

**Reply by givanz** [reply]
Great news! thanks for letting me know.
